lints/lint_issuer_multiple_attr_in_rdn.go

```diff
@@ -0,0 +1,45 @@
+// lint_issuer_multiple_attr_in_rdn.go
+
+package lints
+
+import (
+	"encoding/asn1"
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zcrypto/x509/pkix"
+	"github.com/zmap/zlint/util"
+)
+
+type IssuerRDNHasMultipleAttribute struct {
+	// Internal data here
+}
+
+func (l *IssuerRDNHasMultipleAttribute) Initialize() error {
+	return nil
+}
+
+func (l *IssuerRDNHasMultipleAttribute) CheckApplies(c *x509.Certificate) bool {
+	return true
+}
+
+func (l *IssuerRDNHasMultipleAttribute) RunTest(c *x509.Certificate) (ResultStruct, error) {
+	var issuer pkix.RDNSequence
```

Comment:
So is it the case that anything after the first one is a violation? As it stands it looks like you're checking if each RDN has a length > 1, not the other way around.